var/spack/repos/builtin/packages/fastdfs/fastdfs/package.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Fastdfs(Package):
```

Comment:
This file looks correct but is in the wrong directory.

Reply:
Sorry, I made this mistake again...
I fixed it in https://github.com/spack/spack/pull/14378/commits/cfdacd180dc06b4bf792b02ef3664c181d0c0a00